lib/embed/viewer/common_viewer.rb

```diff
@@ -54,7 +54,11 @@ def footer_html
               end
               if self.is_a?(Embed::Viewer::Image) && !@request.hide_download?
                 doc.button(class: 'sul-embed-footer-tool sul-embed-btn sul-embed-btn-xs sul-embed-btn-default fa fa-download', 'aria-expanded' => 'false', 'data-sul-embed-toggle' => 'sul-embed-download-panel')
-              end
+              end  
+              if external_url.present?              
+               doc.button(class: 'sul-embed-footer-tool sul-embed-btn sul-embed-btn-xs sul-embed-btn-default fa fa-external-link', 'aria-expanded' => 'false', 
+                  onclick:"location.href='#{external_url}'",)
```

Comment:
Unfortunately, the style doesn't work. It shows blank box for the default mode and shows the right behavior with hover mode.